src/field_theory/krull_topology.lean

```diff
@@ -283,3 +285,242 @@ begin
 end
 
 end totally_disconnected
+
+section compact
+
```

Comment:
Could you please add a module docstring that explains/announces what you are going to do in the next 200 lines?